cosmos/__init__.py

```diff
@@ -2,4 +2,4 @@
 Astronomer Cosmos is a library for rendering 3rd party workflows in Airflow.
 """
 
-__version__ = "0.6.7"
+__version__ = "0.7.0rc2"
```

Comment:
It is probably worth having a separate PR with the release tag, so we have independent commits

Reply:
Yeah, just have this in there for now to be able to tag pre-releases for testing.